frontend/pages/_document.jsx

```diff
@@ -52,13 +52,10 @@ class Document extends BaseDocument {
             href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
           />
           {/* Material Design Bootstrap CSS */}
-          <link
```

Comment:
Are we not using mdbootstrap anymore or was it moved somewhere else? In both cases, should we remove it from here?

Reply:
It is not being used anymore; it was overriding styles in Material-UI. I'd prefer to leave it commented out here for now, just to keep track of major changes between the React side and Django side. If we don't encounter any strange styling behavior, we can remove it soon.